### PR TITLE
Redux integration

### PR DESCRIPTION
**Pull request by Logopeddd**

### CODE REVIEW

File: src/components/filter-sidebar/filter-sidebar.js [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import { Sidebar, Box, Label } from 'grommet';
 import FilterSidebarForm from './filter-sidebar_form';
+import { sidebarEnabled } from '../../actions/sidebar';
 
 class FilterSidebar extends React.Component {
+  constructor() {
+    super();
+    this.onClose = this.onClose.bind(this);
```
Comment (aaneitchik) [inline src/components/filter-sidebar/filter-sidebar.js]: I suggest using public class fields syntax:
https://reactjs.org/docs/handling-events.html

It's enabled in create-react-app by default

**Review by aaneitchik — APPROVED** [review]
Of course, there is no right way to do this, but I like this way of structuring redux projects more:
https://marmelab.com/blog/2015/12/17/react-directory-structure.html

Me personally, I like this approach too:
https://github.com/erikras/ducks-modular-redux

Of course, you're free to choose how to structure your project, that's just some things I suggest reading :)

File: src/components/filter-sidebar/filter-sidebar.js (hunk 1 above)
Comment (Logopeddd) [reply src/components/filter-sidebar/filter-sidebar.js]: Should  I use **Experimental** public class fields syntax?

Comment (aaneitchik) [reply src/components/filter-sidebar/filter-sidebar.js]: yes)

**Pull request merged**